general-use.js

```diff
@@ -741,12 +752,17 @@ function parseTengwa(callback, options, tehta, tehtaFrom) {
             };
         } else if (character === "e" && (!tehta || tehta === "a")) { // ae or e after consonants
             return callback(makeColumn("yanta", {from: "e", diphthong: true}), tehta, tehtaFrom);
+        } else if (character === "e" && (!tehta || tehta === "i")) { // ie or e after consonants
+            return callback(makeColumn("yanta", {from: "e", diphthong: true}), tehta, tehtaFrom);
+        } else if (character === "e" && (!tehta || tehta === "o")) { // oe or e after consonants
+            return callback(makeColumn("yanta", {from: "e", diphthong: true}), tehta, tehtaFrom);
         } else if (character === "ë") { // if "ë" makes it this far, it's a diaresis for english
             return callback(makeColumn("short-carrier", {from: ""}).addAbove("e", {from: "e"}));
         } else if (character === "y") {
             return Parser.countPrimes(function (primes) {
                 if (primes === 0) {
-                    return callback(makeColumn("anna", {from: ""}), tehta, tehtaFrom);
+                  return callback(makeColumn("anna", {from: ""}), tehta, tehtaFrom);
```

Comment:
@dreamingfifi Please verify that this is correct. This means that "y" will be cast as "anna" and "y`" will be cast as long carrier with a dot above.

I believe that "anna" is a silent carrier and that we need either an inverted hat above or diaeresis below. Wilya, as mentioned was previously here, was clearly just an error, but the dots were present.

On http://at.mansbjorkman.net/teng_general.htm, it seems that there’s a mode where "y" would be represented with the squiggle above that is usually reserved for "w". We could make that one of the available alternatives.

Reply:
I’m electing to land this without modification. It’ll clear up in the merge with #35.